aa_stripe/management/commands/check_pending_webhooks.py

```diff
@@ -27,18 +29,22 @@ class Command(BaseCommand):
     help = "Check pending webhooks at Stripe API"
 
     def handle(self, *args, **options):
-        stripe.api_key = settings.STRIPE_API_KEY
-        pending_events_threshold = get_pending_events_threshold()
+        stripe.api_key = stripe_settings.API_KEY
+        pending_events_threshold = stripe_settings.PENDING_EVENTS_THRESHOLD
 
         pending_events = []
-        last_event = StripeWebhook.objects.latest("created").id
+        last_event = StripeWebhook.objects.last()
+        last_event_id = last_event.id if last_event else None
         while True:
-            event_list = stripe.Event.list(starting_after=last_event, limit=100)  # 100 is the maximum
-            pending_events += event_list["data"]
+            event_list = stripe.Event.list(ending_before=last_event_id, limit=100)  # 100 is the maximum
+            for event in event_list["data"]:
+                if event["pending_webhooks"] > 0:
+                    pending_events.append(event)
```

Comment:
clearly, this is not pending events amount but amount of pending webhooks for the events